ambry-cloud/src/main/java/com.github.ambry.cloud/azure/AzureCloudConfig.java

```diff
@@ -29,12 +28,20 @@
   public static final String COSMOS_COLLECTION_LINK = "cosmos.collection.link";
   public static final String COSMOS_KEY = "cosmos.key";
   public static final String COSMOS_DIRECT_HTTPS = "cosmos.direct.https";
+  public static final String COSMOS_QUERY_BATCH_SIZE = "cosmos.query.batch.size";
+  public static final String COSMOS_REQUEST_CHARGE_THRESHOLD = "cosmos.request.charge.threshold";
+  public static final String COSMOS_CONTINUATION_TOKEN_LIMIT = "cosmos.continuation.token.limit";
   public static final String AZURE_PURGE_BATCH_SIZE = "azure.purge.batch.size";
   public static final String AZURE_NAME_SCHEME_VERSION = "azure.name.scheme.version";
   public static final String AZURE_BLOB_CONTAINER_STRATEGY = "azure.blob.container.strategy";
   // Per docs.microsoft.com/en-us/rest/api/storageservices/blob-batch
   public static final int MAX_PURGE_BATCH_SIZE = 256;
   public static final int DEFAULT_PURGE_BATCH_SIZE = 100;
+  public static final int DEFAULT_QUERY_BATCH_SIZE = 100;
+  public static final int DEFAULT_COSMOS_MAX_RETRIES = 5;
```

Comment:
Looks like `DEFAULT_COSMOS_MAX_RETRIES` is never used for now. Will it be adopted in future PR?

Reply:
Removed